sites/fast-website/src/app/data/community.data.ts

```diff
@@ -38,6 +38,7 @@ export const communityContentPlacementData: CommunityContentPlacementData[] = [
     },
     {
         actionLink: "https://medium.com/fast-dna",
+        actionText: "Follow us on Medium",
```

Comment:
nit: Could we maybe change this to something like "Read our blog on Medium"? That is the action text on the current website. Follow us sounds more like Twitter than Medium to me.